crowdin_api/api_resources/screenshots/resource.py

```diff
@@ -66,14 +70,19 @@ def add_screenshot(
 
         projectId = projectId or self.get_project_id()
 
+        request_data = {
+            "storageId": storageId,
+            "name": name,
+            "autoTag": autoTag,
+        }
+
+        if labelIds is not None:
```

Comment:
there is no need to check if the parameter exists. Check the similar method - https://github.com/crowdin/crowdin-api-client-python/blob/main/crowdin_api/api_resources/source_files/resource.py#L193